### PR TITLE
Added since, fastpath settings, index settings, streamsnap settings

### PR DESCRIPTION
**Pull request by patrickhargett**

### CODE REVIEW

**Review by mbreitung — DISMISSED** [review]

File: support_bundle/get_support_bundle.sh [hunk 1]
```diff
@@ -145,29 +154,39 @@ do
     kubectl ${KUBE_OPTS} exec -it $pod -c cassandra -- nodetool compactionstats | tee -a ${LOG_DIR}/cassandra/$pod/nodetool_compactionstats.log
 done
 
-echo "Fetch Elasticsearch health info"
+#Fetch Elasticsearch health info
```
Comment (mbreitung) [inline support_bundle/get_support_bundle.sh:157]: this line can be removed

File: support_bundle/get_support_bundle.sh [hunk 2]
```diff
@@ -145,29 +154,39 @@ do
...
+echo "Fetching Elasticsearch health info"
 mkdir -p ${LOG_DIR}/elasticsearch
 printf "Pod#\tFilesystem\tSize\tUsed\tAvail\tUse\tMounted on\n" | tee -a ${LOG_DIR}/elasticsearch/elasticsearch_storage.log
+# CHECK HERE IF THE TLS ENV VARIABLE IS SET IN ELASTICSEARCH, AND BUILD THE CURL COMMAND OUT
+ELASTIC_POD=$(kubectl ${KUBE_OPTS} get po -l role=elasticsearch --no-headers | head -1 | awk '{print $1}')
+ELASTIC_TLS=$(kubectl ${KUBE_OPTS} exec -it ${ELASTIC_POD} -- env | grep -i ELASTICSEARCH_TLS_ENCRYPTION)
+
+if [[ ${ELASTIC_TLS} == *"ELASTICSEARCH_TLS_ENCRYPTION=true"* ]]; then
+    ELASTIC_CURL='curl --cacert /usr/share/elasticsearch/config/root-ca.pem https://${ELASTICSEARCH_ADMINUSER}:${ELASTICSEARCH_ADMIN_PASSWORD}'
```
[the remark this answers is not in the document]
Comment (patrickhargett) [reply support_bundle/get_support_bundle.sh]: @mbreitung added -s to the curl command in the latest commit.

File: support_bundle/get_support_bundle.sh (hunk 1 above)
Comment (patrickhargett) [reply support_bundle/get_support_bundle.sh]: @mbreitung removed that line.

**Review by mbreitung — APPROVED** [review]

**Review by mbreitung — APPROVED** [review]

**Review by hectorgarzon — APPROVED** [review]

**Comment by patrickhargett** [comment]
@mbreitung Can we merge this please?

**Pull request merged**